main.go

```diff
@@ -56,7 +58,7 @@ func main() {
 	command := flag.Arg(0)
 	switch {
 	case command == "update":
-		username, token, pemPath, pemPwd = checkEnvs(username, token, pemPath, pemPwd)
+		username, token, pemPath, pemPwd := checkEnvs(user, token, systemUser, pemPath, pemPwd)
```

Comment:
What's the reasoning to merge user and systemUser as username within the checkEnvs func?